tests/unit/test_images.py

```diff
@@ -0,0 +1,148 @@
+import os
+import uuid
+from unittest import mock
+
+import jsonschema
+import pytest
+
+from app.services.images import ImageService
+
+TEST_IMAGE = "test_cat.jpg"
+TEST_IMAGE_PATH = f"{os.path.dirname(os.path.realpath(__file__))}/assets/{TEST_IMAGE}"
+TEST_UUID = uuid.UUID("c0ff33c0-ffee-c0ff-eec0-ffeec0ffeec0")
+QUERY = "flowers"
+
+DOWNLOADED_PATH = ImageService.get_downloaded_path()
+DOWNLOADED_FILE_PATH = f"{DOWNLOADED_PATH}/{QUERY}/{str(TEST_UUID)}.jpg"
+UPLOAD_PATH = ImageService.get_upload_path()
+UPLOAD_FILE_PATH = f"{UPLOAD_PATH}/{TEST_IMAGE}"
+
+
+# Validation Schemas
+model_prediction_schema = {
```

Comment:
by choice schemas exist as jsonschemas (instead of reusing the API's schemas). I think this is an important distinction for testing, where our tests in part validate the API contract.